Mlem/App/Views/Root/Tabs/Feeds/FeedsView.swift

```diff
@@ -80,11 +89,24 @@ struct FeedsView: View {
             api: AppState.main.firstApi,
             feedType: initialFeedSelection.associatedApiType
         ))
+        if let firstUser = AppState.main.firstAccount as? UserAccount {
+            _savedFeedLoader = .init(wrappedValue: .init(
+                api: AppState.main.firstApi,
+                userId: firstUser.id,
+                sortType: .new,
+                savedOnly: true,
+                smallAvatarSize: AppConstants.smallAvatarSize,
+                largeAvatarSize: AppConstants.largeAvatarSize
+            ))
+            _feedOptions = .init(wrappedValue: FeedSelection.allCases)
+        } else {
+            _feedOptions = .init(wrappedValue: FeedSelection.guestCases)
```

Comment:
This value isn't updated when the active account changes. This either needs to be updated via an `.onChange` when the active account changes, or be a computed property rather than an `@State` property.

Reply:
Nice catch, that functionality got eaten by the merge 😶